src/main/java/seedu/mark/logic/commands/GotoCommand.java

```diff
@@ -44,7 +45,8 @@ public CommandResult execute(Model model, Storage storage) throws CommandExcepti
         Bookmark bookmarkToOpen = lastShownList.get(targetIndex.getZeroBased());
         model.setCurrentUrl(bookmarkToOpen.getUrl());
 
-        return new CommandResult(String.format(MESSAGE_GOTO_BOOKMARK_ACKNOWLEDGEMENT, bookmarkToOpen));
+        return new TabCommandResult(String.format(MESSAGE_GOTO_BOOKMARK_ACKNOWLEDGEMENT, bookmarkToOpen),
```

Comment:
Can make GotoCommandResult? Quite weird to return a TabCommandResult.

Reply:
Actually there is no need for gotoCommand to switch tab alr bc in the bookmarkList panel, whenever url changes (either from gotocommand or GUI selection), it will automatically change to online tab.